contracts/cw-ics721-bridge/src/msg.rs

```diff
@@ -135,6 +135,40 @@ pub enum QueryMsg {
     /// Gets this contract's cw721-proxy if one is set.
     #[returns(Option<::cosmwasm_std::Addr>)]
     Proxy {},
+
+    /// Gets the code used for instantiating new cw721s.
+    #[returns(u64)]
+    Cw721CodeId {},
+
+    /// Gets a list of classID (from NonFungibleTokenPacketData) and
+    /// cw721 contract we have instantiated for that classID.
+    #[returns(Vec<ClassIdToNftContractResponse>)]
+    ClassIdToNftContract {},
+
+    /// Gets a list of class ID, token ID, and local channel ID. Used
+    /// to determine the local channel that NFTs have been sent
+    /// out on.
+    #[returns(Vec<ClassTokenToChannelResponse>)]
+    OutgoingClassTokenToChannel {},
+
+    /// Gets a list of class ID, token ID, and local channel ID. Used
+    /// to determine the local channel that NFTs have arrived at
+    /// this contract.
+    #[returns(Vec<ClassTokenToChannelResponse>)]
+    IncomingClassTokenToChannel {},
```

Comment:
what do you imagine these queries being used for? I wonder if it would be more useful to have a query for a particular `(class, token)` pair to get the channel it has gone out / in for.

Reply:
Collections are 'multi-chain-enabled' by ICS721. So marketplaces and other tools will need multi-chain utilities. Like marketplaces, relayers and other upcoming multi-chain tools, it will be useful to know:
- where have NFTs been send to/from (incoming/outgoing)
- what collections have been created for each ICS721 contract